modules/javafx.web/src/test/resources/test/html/pointerevents-stroke.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Test pointer events on stroke</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style type="text/css">
+            rect, path, polyline, line, circle, ellipse, polygon {
+                pointer-events: stroke;
+                stroke: gray;
+                stroke-opacity: 1;
+                stroke-width: 10px;
+                fill: transparent;
+            }
+            #dashed {
+                stroke-dasharray: 20px;
+                stroke-dashoffset: 20px;
+            }
+            rect.clicked, path.clicked, polyline.clicked, line.clicked, circle.clicked, ellipse.clicked, polygon.clicked {
+                stroke: green;
+            }
+        </style>
+        <script>
+            var testElements = [ "polyline", "path", "rect", "circle", "ellipse", "polygon", "dashed" ];
+            window.addEventListener( 'DOMContentLoaded', ( event ) => {
+                for ( var el of testElements ) {
+                    document.getElementById( el ).addEventListener( "mousedown", function ( evt ) {
```

Comment:
Remove extra space before and after the parameter.
```
window.addEventListener('DOMContentLoaded', (event) => {
```

Ditto for the whole file.